orchestrator/nodeManager.js

```diff
@@ -23,12 +23,53 @@ var config = require("./config");
 var dojotModule = require("@dojot/dojot-module");
 var dojotConfig = dojotModule.Config;
 
+var MongoManager = require('./mongodb');
+
 class NodeManager {
   constructor() {
     this.nodes = {};
   }
 
+  upContainers(tenant) {
```

Comment:
I'd change this method to a verb, such as 'startContainer' (same for mongoConnection, for instance, 'createMongoConnection')